packages/token-lists/lists/amplitude.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "amplitude",
+  "timestamp": "2023-03-31T12:02:57.229Z",
+  "tokens": [
+    {
+      "networkId": 200,
+      "address": "2124-0-0",
+      "parachainId": 2124,
+      "ethereumChainId": 2124,
+      "assetType": 0,
+      "assetIndex": 0,
+      "symbol": "AMPE",
+      "decimals": 12,
+      "name": "Amplitude"
+    },
+    {
+      "networkId": 200,
+      "address": "2124-2-516",
+      "parachainId": 2124,
+      "assetType": 2,
+      "assetIndex": 0,
+      "symbol": "KSM",
+      "decimals": 12,
+      "name": "Kusama"
+    }
```

Comment:
So this is correct, right @RustNinja?
Do you know what the `networkId` is supposed to be?

Reply:
(Sorry, I changed the "address" to "2124-2-0" now and force-pushed again)

Reply:
@ebma 
https://github.com/paritytech/polkadot/blob/master/xcm/src/v3/junction.rs#L40-L65
or
some polkadot[.].js.org network id

Reply:
@ebma seems 200 is network id for kusama
and 300 is network id for polkadot

Reply:
Oh okay yeah maybe that's it